### PR TITLE
Including i18n in biocache: 1. JS files related with Leaflet map

### PR DESCRIPTION
**Pull request by santiagomrv**
Included Jquery_i18n module, new message.properties and new vars in the JS files related with Leaflet-Map view.

### CODE REVIEW

File: web-app/js/leaflet-plugins/draw/leaflet.draw-src.js [hunk 1]
```diff
@@ -6,103 +6,115 @@
...
+				actions: {
+					title: jQuery.i18n.prop('advancedsearch.js.map.canceldrawing'),
+					text: jQuery.i18n.prop('advancedsearch.js.map.cancel')
+				},
+				undo: {
+					title: jQuery.i18n.prop('advancedsearch.js.map.deletepointdrawn'),
+					text: jQuery.i18n.prop('advancedsearch.js.map.deletepoint')
+				},
+				buttons: {
+					polyline: 'Draw a polyline',
```
Comment (djtfmartin) [inline web-app/js/leaflet-plugins/draw/leaflet.draw-src.js]: Do we need to i18n this one ?

File: web-app/js/leaflet-plugins/draw/leaflet.draw-src.js [hunk 2]
```diff
@@ -6,103 +6,115 @@
...
+					polygon: jQuery.i18n.prop('advancedsearch.js.map.polygon'),
+					rectangle: jQuery.i18n.prop('advancedsearch.js.map.rectangle'),
+					circle: jQuery.i18n.prop('advancedsearch.js.map.circle'),
+					marker: 'Draw a marker'
```
Comment (djtfmartin) [inline web-app/js/leaflet-plugins/draw/leaflet.draw-src.js]: Do we need to i18n this one ?

File: web-app/js/leaflet-plugins/draw/leaflet.draw-src.js [hunk 3]
```diff
@@ -6,103 +6,115 @@
...
-					text: 'Save'
+			handlers: {
+				circle: {
+					tooltip: {
+						start: jQuery.i18n.prop('advancedsearch.js.map.circle.tooltip')
+					}
+				},
+				marker: {
+					tooltip: {
+						start: 'Click map to place marker.'
```
Comment (djtfmartin) [inline web-app/js/leaflet-plugins/draw/leaflet.draw-src.js]: Do we need to i18n this one ?

File: web-app/js/leaflet-plugins/draw/leaflet.draw-src.js (hunk 1 above)
Comment (santiagomrv) [inline web-app/js/leaflet-plugins/draw/leaflet.draw-src.js]: Done

File: web-app/js/leaflet-plugins/draw/leaflet.draw-src.js (hunk 2 above)
Comment (santiagomrv) [inline web-app/js/leaflet-plugins/draw/leaflet.draw-src.js]: Done

**Pull request merged**